### ISSUE TITLE
open tasks in tray icon caused FS to crash

### ISSUE THREAD

**Issue by u-Sir** [adirh3/Fluent-Search#724]
**Describe the bug**
open tasks in tray icon caused FS to crash. but search `task` and open is ok.

**To Reproduce**
1. right click on FS tray icon
2. Click on 'tasks'

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Window 11 Version: 22h2
 - Fluent Search Version: 0.9.92.63
 - Installation source: Installer

**Additional context**
```
Application: FluentSearch.exe
CoreCLR Version: 7.0.22.51805
.NET Version: 7.0.0
Description: The process was terminated due to an unhandled exception.
Exception Info: System.MissingMethodException: Method not found: 'Void Avalonia.AvaloniaObject.RaisePropertyChanged(Avalonia.AvaloniaProperty`1<!!0>, Avalonia.Data.Optional`1<!!0>, Avalonia.Data.BindingValue`1<!!0>, Avalonia.Data.BindingPriority)'.
   at Avalonia.Controls.PanAndZoom.ZoomBorder.InvalidateProperties()
   at Avalonia.Controls.PanAndZoom.ZoomBorder.Invalidate(Boolean skipTransitions) in C:\Git\PanAndZoom\src\Avalonia.Controls.PanAndZoom\ZoomBorder.cs:line 290
   at Avalonia.Controls.PanAndZoom.ZoomBorder.SetMatrix(Matrix matrix, Boolean skipTransitions) in C:\Git\PanAndZoom\src\Avalonia.Controls.PanAndZoom\ZoomBorder.cs:line 376
   at Avalonia.Controls.PanAndZoom.ZoomBorder.ResetMatrix(Boolean skipTransitions) in C:\Git\PanAndZoom\src\Avalonia.Controls.PanAndZoom\ZoomBorder.cs:line 395
   at Avalonia.Controls.PanAndZoom.ZoomBorder.ResetMatrix() in C:\Git\PanAndZoom\src\Avalonia.Controls.PanAndZoom\ZoomBorder.cs:line 386
   at Blast.Views.TasksWindow.<>c__DisplayClass11_0.<FocusSelectedTaskProject>b__0() in D:\a\1\s\Blast\Views\TasksWindow.xaml.cs:line 113
   at Blast.Views.TasksWindow.FocusSelectedTaskProject() in D:\a\1\s\Blast\Views\TasksWindow.xaml.cs:line 111
   at Blast.Views.TasksWindow..ctor() in D:\a\1\s\Blast\Views\TasksWindow.xaml.cs:line 47
   at Blast.WindowCreator.CreateNewWindow(String name) in D:\a\1\s\Blast\WindowCreator.cs:line 81
   at Blast.WindowCreator.<>c__DisplayClass1_0.<CreateWindow>b__0() in D:\a\1\s\Blast\WindowCreator.cs:line 55
   at Blast.API.Core.UI.AvaloniaDispatcherWrapper.InvokeAsync[TResult](Func`1 function, DispatcherPriority priority, CancellationToken cancellationToken) in D:\a\1\s\Blast.API.Core\UI\AvaloniaDispatcherWrapper.cs:line 62
   at Blast.WindowCreator.CreateWindow(String name) in D:\a\1\s\Blast\WindowCreator.cs:line 54
   at Blast.WindowCreator.GetOrCreate(String name) in D:\a\1\s\Blast\WindowCreator.cs:line 47
   at Blast.WindowCreator.CreateAndShow(String name, ShowWindowOptions showWindowOptions) in D:\a\1\s\Blast\WindowCreator.cs:line 65
   at Blast.Views.SystemTrayWindow.<>c.<.ctor>b__5_7() in D:\a\1\s\Blast\Views\SystemTrayWindow.xaml.cs:line 109
   at Blast.Views.SystemTrayItem.Command() in D:\a\1\s\Blast\Views\SystemTrayWindow.xaml.cs:line 47
   at CompiledAvaloniaXaml.XamlIlTrampolines.FluentSearch:Blast.Views.SystemTrayItem+Command_0!CommandExecuteTrampoline(Object, Object)
   at Avalonia.Markup.Xaml.MarkupExtensions.CompiledBindings.CommandAccessorPlugin.CommandAccessor.Command.Execute(Object parameter)
   at Avalonia.Controls.Button.OnClick()
   at Avalonia.Controls.Button.OnPointerReleased(PointerReleasedEventArgs e)
   at Avalonia.Input.InputElement.<>c.<.cctor>b__31_9(InputElement x, PointerReleasedEventArgs e)
   at System.Reactive.Subjects.Subject`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/Subjects/Subject.cs:line 141
   at Avalonia.Interactivity.EventRoute.RaiseEventImpl(RoutedEventArgs e)
   at Avalonia.Interactivity.Interactive.RaiseEvent(RoutedEventArgs e)
   at Avalonia.Input.MouseDevice.MouseUp(IMouseDevice device, UInt64 timestamp, IInputRoot root, Point p, PointerPointProperties props, KeyModifiers inputModifiers, IInputElement hitTest)
   at Avalonia.Input.MouseDevice.ProcessRawEvent(RawPointerEventArgs e)
   at Avalonia.Controls.TopLevel.HandleInput(RawInputEventArgs e)
   at Avalonia.Win32.WindowImpl.AppWndProc(IntPtr hWnd, UInt32 msg, IntPtr wParam, IntPtr lParam)
   at Avalonia.Win32.WindowImpl.WndProc(IntPtr hWnd, UInt32 msg, IntPtr wParam, IntPtr lParam)
   at Avalonia.Win32.Interop.UnmanagedMethods.DispatchMessage(MSG& lpmsg)
   at Avalonia.Win32.Win32Platform.RunLoop(CancellationToken cancellationToken)
   at Avalonia.Threading.Dispatcher.MainLoop(CancellationToken cancellationToken)
   at Avalonia.Controls.ApplicationLifetimes.ClassicDesktopStyleApplicationLifetime.Start(String[] args)
   at Blast.Program.Main(String[] args) in D:\a\1\s\Blast\Program.cs:line 301
```

**Comment by adirh3** [comment]
@u-Sir same with latest version?

**Comment by u-Sir** [comment]
> @u-Sir same with latest version?

no. but tasks window does not response any action, and cause FS mess up
![图片](https://user-images.githubusercontent.com/93647031/206480470-8fec57ae-84ad-44cb-9e6b-0af13d8e9032.png)

**Comment by u-Sir** [comment]
@adirh3 I found open tasks window from tray icon will cause FS no response, and it seems cause #729 #726

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 0.9.92.66                                         . Changelog - https://install.appcenter.ms/users/adirh3-gmail.com/apps/fluent-search-alpha/distribution_groups/exe
